rest-service/manager_rest/blueprints_manager.py

```diff
@@ -61,6 +60,9 @@ def deployments_list(self, include=None, filters=None, pagination=None):
         return self.sm.deployments_list(include=include, filters=filters,
                                         pagination=pagination)
 
+    def snapshots_list(self, include=None):
```

Comment:
This call should support pagination, please look here:
https://github.com/cloudify-cosmo/cloudify-manager/pull/322/files